CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Fixed custom sorting Not Updated Until Restart the Application in favorites tab
+
```

Comment:
What happened to the crash fix entry from the main branch?

https://github.com/FossifyOrg/Phone/blob/main/CHANGELOG.md#fixed